unit/constant/generate_constants.go

```diff
@@ -0,0 +1,327 @@
+// Copyright ©2014 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"go/format"
+	"log"
+	"math"
+	"os"
+	"strings"
+	"text/template"
+
+	"gonum.org/v1/gonum/unit"
+)
+
+const (
+	elementaryCharge = 1.6021766208e-19
+	fineStructure    = 7.2973525664e-3
+	lightSpeed       = 2.99792458e8
+	planck           = 6.62607015e-34
+)
+
+var constants = []Constant{
+	{
+		Name: "AtomicMass", Value: 1.660539040e-27,
+		Dimensions: []Dimension{{massName, 1}},
+		Comment:    "AtomicMass is the atomic mass constant (mᵤ), one twelfth of the mass of an unbound atom of carbon-12 at rest and in its ground state.",
+	},
+	{
+		Name: "Avogadro", Value: 6.02214076e23,
+		Dimensions: []Dimension{{moleName, -1}},
+		Comment:    "Avogadro is the Avogadro constant (A), the number of constituent particles contained in one mole of a substance.",
```

Comment:
I think the constants that are defined should have a note that they are so (i.e. actually constant, at least until the next SI revision in 50 years)

Reply:
I will add this for defined constants.